VultisigApp/VultisigApp/Utils/Utils.swift

```diff
@@ -225,29 +225,61 @@ enum Utils {
         return Image(cgImage, scale: 1.0, orientation: .up, label: Text("QRCode"))
     }
     
-    public static func parseCryptoURI(_ uri: String) -> (address: String, amount: String, message: String){
+    public static func parseCryptoURI(_ uri: String) -> (address: String, amount: String, message: String) {
         
         var address: String = .empty
         var amount: String = .empty
         var message: String = .empty
         
-        guard let url = URLComponents(string: uri) else {
-            print("Invalid URI")
-            return (.empty, .empty, .empty)
-        }
-        
-        address = url.host ?? url.path
-        
-        url.queryItems?.forEach { item in
-            switch item.name {
-            case "amount":
-                amount = item.value ?? ""
-            case "label", "message":
-                if let value = item.value, !value.isEmpty {
-                    message += (message.isEmpty ? "" : " ") + value
+        if uri.hasPrefix("ton://") {
+            // Manipulação especial para URIs que começam com "ton://"
```

Comment:
don't put non-english comments in the code base please

Reply:
Removed